.github/workflows/tox-pytest.yml

```diff
@@ -59,7 +59,7 @@ jobs:
           conda run -n ferc1_eia_match tox
 
       - name: Upload test coverage report to CodeCov
-        uses: codecov/codecov-action@3.1.2
+        uses: codecov/codecov-action@v3.1.2
```

Comment:
@zaneselvans any idea why this needs to be `v3.1.2` for the CI to pass but dependabot made it `3.1.2` in #50 ?

Reply:
Hmm, dunno. That sounds buggy to me.